### DISCUSSION TITLE
Cybersecurity eBook Finished

### DISCUSSION THREAD

**Discussion by jimmarq** [Mogztter/asciidoctor-web-pdf#668] [Show and tell]
Thanks for this project! It produced a PDF that passed all accessibility requirements for my institution.

Here's a link to the repository: https://github.com/jimmarq/cyfunbook

Here's a direct link to the PDF: https://github.com/jimmarq/cyfunbook/releases/download/v1.0/Cybersecurity_Fundamentals-1.0.pdf

**Comment by ggrossetie** [comment]
Thanks for sharing and congrats! 👍🏻

**Comment by silopolis** [comment]
Le mar. 4 oct. 2022 à 18:53, Jim Marquardson ***@***.***> a
écrit :

> Thanks for this project! It produced a PDF that passed all accessibility
> requirements for my institution.
>
> Here's a link to the repository: https://github.com/jimmarq/cyfunbook
>
> Here's a direct link to the PDF:
> https://github.com/jimmarq/cyfunbook/releases/download/v1.0/Cybersecurity_Fundamentals-1.0.pdf
>
Gorgeous and surely interesting... Added to bookshelf !
Thanks a lot for sharing, this is interesting wrt to our future plans for
LinuxCNC documentation...

TY
